src/Proyecto26.RestClient/Helpers/HttpBase.cs

```diff
@@ -46,15 +46,15 @@ public static IEnumerator CreateRequestAndRetry(RequestHelper options, Action<Re
                         callback(null, response);
                         break;
                     }
-                    else if (!options.IsAborted && retries < options.Retries && (IsNetworkError))
+                    else if (!options.IsAborted && retries < options.Retries && (!options.RetryCallbackOnlyOnNetworkErrors || request.isNetworkError))
```

Comment:
Hello mate, can you use `IsNetworkError` instead of `request.isNetworkError` to support new versions of Unity, please?

Reply:
sure! sorry I didn't realize about it. Now it's changed